defaults/perftune_results.json

```diff
@@ -0,0 +1,126 @@
+{
+  "aws":
+  {
+    "x86_64": {
+      "2": {
+        "get-cpu-mask": "0x00000003",
+        "get-irq-cpu-mask": "0x00000003",
+        "dump-options-file": {"cpu_mask": "0x00000003", "irq_core_auto_detection_ratio": 16, "irq_cpu_mask": "0x00000003", "tune": ["net"]}
```

Comment:
if all entries here have the exact same `irq_core_auto_detection_ratio": 16`, why do we need to repeat it everywhere? couldn't we have it fixed somewhere?

Reply:
True, tune is constant too. I'll add them in the code instead.